components/scream/tests/coupled/physics_only/shoc_cld_spa_p3_rrtmgp/shoc_cld_spa_p3_rrtmgp.cpp

```diff
@@ -0,0 +1,64 @@
+#include <catch2/catch.hpp>
+
+// Boiler plate, needed for all runs
+#include "control/atmosphere_driver.hpp"
+#include "share/atm_process/atmosphere_process.hpp"
```

Comment:
You are not explicitly using anything from this header, so this include can be removed.